frontend/src/Components/Header.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+const Header = () => {
+  return (
+    <header className="header">
+      <nav className="nav">
+        <a href="/">
+          <div className="nav__logo">
+            <img src="./images/logo.png" alt="logo" />
+          </div>
+        </a>
+        <div className="nav__links">
+          <div className="nav__search">
+            <input type="text" placeholder="Search" />
+            <button>
+              <i className="fas fa-search" />
+            </button>
+          </div>
+          <a href="./product.html" className="nav__link">
+            <i className="fa-solid fa-crown" />
+            <span className="nav__link--text">Products</span>
+          </a>
+          <a href="./wishlist.html" className="nav__link">
```

Comment:
suggestion - replace `<a>` with `<Link>`

Reply:
Yes I am aware of these will do it soon.